playground/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "playground",
```

Comment:
`@outerbase/sdk/playground`

Reply:
ah interesting
well then, you should add `private: true` so it can't be published :P